dask/dataframe/shuffle.py

```diff
@@ -208,7 +219,10 @@ def set_partition(
             column_dtype=df.columns.dtype,
         )
 
-    df4.divisions = divisions
+    if hasattr(divisions, "tolist"):
+        df4.divisions = divisions.tolist()
+    else:
+        df4.divisions = divisions.values.tolist()
```

Comment:
Do cudf series objects not have a `tolist()` method?  If not, can we add it there instead?

Reply:
Right, no `tolist()` - But you are right, I can certainly prepare a small PR to propose this change.

Reply:
Just a not: List comprehension over `divisions.values` takes care of the problems I was seeing for `divisions.values.tolist()` with timestamps - Shouldn't need `tolist()` anymore.

Reply:
Glad to hear it.  It would still be nice to raise it as an issue.  It'll linger around for a while but maybe some drive-by developer will take it up.  It's always nice to have easy to fix issues.